docs/series/LAMP-server.md

```diff
@@ -0,0 +1,241 @@
+
+| Name | Description |
+|----------|----------|
+| (**L**) AlmaLinux    | The operating system, in this case AlmaLinux 8   |
+| **A**pache 2   | A trusted open-source web server software known for its reliability and scalability.  |
+| **M**ariaDB  | A popular open-source database system for managing structured data efficiently.   |
+| **p**hp-fpm   | A widely-used web scripting language known for its simplicity, flexibility, and strong community.   |
+
+### Requirements
+
+* A privileged user (*sudoer*) or access to the root user.
+* RHEL based OS, AlmaLinux 8 or 9.
+
+## First step - install Apache 2
+
+Apache is a project designed to develop and maintain an open source HTTP server for modern operating systems along with high security with efficient resource consumption and meeting HTTP standards.
+It first appeared in 1995 and since 1996 the most popular web server in the world.
+
+* **Install Apache 2:**
+    ```sh
+    sudo dnf install -y httpd
+    ```
+
+* **Allow traffic on port 80:**
+    ```sh
+    sudo firewall-cmd --add-port=80/tcp --permanent
+    sudo firewall-cmd --reload
+    ```
+
+* **Enable and start apache 2:**
+    ```sh
+    sudo systemctl enable httpd
+    sudo systemctl start httpd
+    ```
+
+You can test that Apache runs properly by accessing http://YOUR-IP (**YOUR-IP** being your ip address), you should get the following page:
+
+![AlmaLinux-Apache-Default-page](/images/HowTo-LAMP-server-1.png)
+
+## Second step - install MariaDB
+
+MariaDB is one of the major relational databases in the market, built by the developers of MySQL, ensuring to remain open-source, with a large part of the cloud utilizing it for data management purposes.
+
+
+* **To use a newer repo of MariaDB (To install a newer version) run the following command in your cmd:**
+    ```sh
+    curl -LsS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | sudo bash -s -- --mariadb-server-version=11.2
+    ```
+    You should get:
+    ```sh
+    # [info] Checking for script prerequisites.
+    # [info] MariaDB Server version 11.2 is valid
+    # [info] Repository file successfully written to /etc/yum.repos.d/mariadb.repo
+    # [info] Adding trusted package signing keys...
+    /etc/pki/rpm-gpg /var/lib/httpd
+    /var/lib/httpd
+    # [info] Successfully added trusted package signing keys
+    # [info] Cleaning package cache...
+    60 files removed
+    ```
+
+* **Then install, enable and start MariaDB:**
+    ```sh
+    sudo dnf install -y MariaDB-server
+    sudo systemctl enable mariadb
+    sudo systemctl start mariadb
+    ```
+
+* **Next, run the mariadb-secure-installation:**
+    ```sh
+    mariadb-secure-installation
+    ```
+
+    Enter the following input:
+
+    | Step | Input |
+    |----------|----------|
+    | Enter current password for root (enter for none) | Enter a new password for MySQL root user |
+    | Switch to unix_socket authentication \[Y/n\]  | N   |
+    | Change the root password? \[Y/n\] | N |
```

Comment:
What's the reasoning behind not using the suggested defaults from `mariadb-secure-installation`?

Reply:
No particular reason, this is so people with less experience can follow those instructions without having doubts. (I'm not sure, should I remove this section?)

Reply:
I like the table so people can follow along. I was curious as to why you deviated from the default suggestions (EX, socket/change root PW). Unless there is some reasoning behind it, I'd defer to the default values. Your call there :).

Reply:
I wasn't sure what I was thinking so I installed mariadb again and ran the secure installation,
seems like I just copied what they suggested:
```
Setting the root password or using the unix_socket ensures that nobody
can log into the MariaDB root user without the proper authorisation.

You already have your root account protected, so you can safely answer 'n'.

Switch to unix_socket authentication [Y/n] 
```
```
You already have your root account protected, so you can safely answer 'n'.

Change the root password? [Y/n] 
```
I see why those are not the best choices now :smile:, I most have been in a hurry when I wrote that part, I'll change it.